templates/layouts/map-section.php

```diff
@@ -0,0 +1,9 @@
+<main>
+    <div class="mapouter">
+        <div class="gmap-canvas">
+            <iframe width="600" height="500" id="gmap_canvas" src="https://maps.google.com/maps?q=university%20of%20san%20francisco&t=&z=13&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0">
+            </iframe>
+            <a href="https://www.whatismyip-address.com/divi-discount/"></a>
```

Comment:
This a tag is empty , so we can remove it

Reply:
okay

Reply:
ill get it done